pino/pino.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for pino v1.0.1
+// Project: https://github.com/mcollina/pino.git
+// Definitions by: Peter Snider <https://github.com/psnider>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts"/>   // only for require()
+
+declare module 'pino' {
+
+    import stream = require('stream')
+
+    interface Serializers {
+        req(req: Object): Object
```

Comment:
actually not. this has a well defined shape see https://github.com/mcollina/pino#pinostdserializersreq